www/index.php

```diff
@@ -72,6 +76,7 @@
     <head>
         <title>WebPagetest - Website Performance and Optimization Test</title>
         <?php $gaTemplate = 'Main'; include ('head.inc'); ?>
+        <script src='https://www.google.com/recaptcha/api.js'></script>

```

Comment:
The recaptcha api code should only be loaded if the setting is present.

Reply:
ok today  i fill in settings.ini